hugo/content/blog/2022-01-16-mumble-1-4-230.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Mumble 1.4.230
+author: Robert Adam
+date: 2022-01-16
+categories:
+  - Release
+---
+
+We are proud to present you with the first stable release of the Mumble 1.4.x release series, which brings many new features, bug fixes and general
+improvements.
+
+You can download the new version from our [**Downloads page**]({{< relref "/downloads" >}}) or the
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.4.230) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+# Mumble 1.4.230
+
+It has been over two years since we released Mumble 1.3.0. Since then many new features and improvements have found their way into the software. And
+not less importantly: many fixes for issues that have been encountered with Mumble. Talking about each and every change here would definitely exceed
+the size of any reasonable release announcement. Therefore, we will restrict ourselves to highlighting what we think are the most prominent and/or
+interesting changes and we invite everyone to simply try out the new version and explore what has changed yourself. For the interested reader, we of
+course also include a full changelog at the bottom of this page as well.
+
+If you are wondering, where versions 1.4.0 to 1.4.229 went, please have a look at a recent
+[blog post]({{< relref "/blog/2021-12-07-new-versioning-scheme/index.md" >}}) of us. In their we go into the details of our _new versioning scheme_ that
+explains the absence of the mentioned version numbers.
+
+
+## Feature spotlight
+
+As there are many, many changes with this new version, here is a selection of things that we think are most prominent. If you're interested in all
+changes, have a look at the full [changelog](#changelog).
```

Comment:
```suggestion
changes, have a look at the [full changelog](#changelog).
```

*I was thinking of suggesting "further down" or similar, but decided against it.*